src/examples/libpmemobj/array/array.c

```diff
@@ -94,7 +101,8 @@ get_type(const char *type_name)
 {
 	const char *names[UNKNOWN_ARRAY_TYPE] = {"int", "PMEMoid", "TOID"};
 	enum array_types type;
-	for (type = 0; type < UNKNOWN_ARRAY_TYPE; type++) {
+	for (type = (enum array_types)0; type < UNKNOWN_ARRAY_TYPE;
```

Comment:
ARRAY_TYPE_UNKNOWN (notice the order of words) should be the value 0 and then this for should look like this: for (type = ARRAY_TYPE_UNKNOWN +1; ...)